pocs/utils/images/__init__.py

```diff
@@ -87,12 +90,39 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
 def _make_pretty_from_fits(fname, **kwargs):
     config = load_config()
 
-    title = '{} {}'.format(kwargs.get('title', ''), current_time().isot)
+    title = '{} {} {}'.format(kwargs.get('title', ''), current_time().isot, 'WCS') # is WCS the coordinate system?
 
     new_filename = fname.replace('.fits', '.jpg')
 
     data = getdata(fname)
-    plt.imshow(data, cmap='cubehelix_r', origin='lower')
+
+    norm = ImageNormalize(interval=MinMaxInterval(), stretch=LogStretch())
+
+    wcs = WCS(fname)
+
+    if wcs.is_celestial:
+        ax = plt.subplot(projection=wcs)
+
+        ax.coords.grid(True, color='white', ls='solid')
+
+        ra = ax.coords[0]
+        dec = ax.coords[1]
+
+        ra.set_axislabel('Right Ascension')
+        dec.set_axislabel('Declination')
+
+        ra.set_major_formatter('hh:mm:ss')
+        dec.set_major_formatter('dd:mm:ss')
```

Comment:
For Huntsman, PANOPTES or in fact most other optical instruments the field of view will be large enough that including seconds in the major tick labels is unnecessary. `hh:mm` and `dd:mm` is probably better.